### PR TITLE
feat: upgrade pack CLI dependency

### PR DESCRIPTION
**Pull request by kadern0**
Fixes: #7337 

**Description**
Seems to be working as expected:
```
skaffold/examples/buildpacks$ ../../out/skaffold build
Generating tags...
 - skaffold-buildpacks -> skaffold-buildpacks:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks: Found. Tagging


skaffold/examples/buildpacks$ ../../out/skaffold dev
Listing files to watch...
 - skaffold-buildpacks
Generating tags...
 - skaffold-buildpacks -> skaffold-buildpacks:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks: Found. Tagging
Tags used in deployment:
 - skaffold-buildpacks -> skaffold-buildpacks:723370b1e06db3208b73fb741ab36545deceb33ebb471e6b5d770ce61da3eada
Starting deploy...
 - service/web configured
 - deployment.apps/web configured
Waiting for deployments to stabilize...
 - deployment/web is ready.
Deployments stabilized in 2.149 seconds
Press Ctrl+C to exit
Watching for changes...
[web] 2022/05/23 23:06:44 Listening on port 8080
^CCleaning up...
 - No resources found


skaffold/examples/buildpacks-java$ ../../out/skaffold build
Generating tags...
 - skaffold-buildpacks -> skaffold-buildpacks:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks: Found. Tagging


skaffold/examples/buildpacks-java$ ../../out/skaffold dev
Listing files to watch...
 - skaffold-buildpacks
Generating tags...
 - skaffold-buildpacks -> skaffold-buildpacks:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks: Found. Tagging
Tags used in deployment:
 - skaffold-buildpacks -> skaffold-buildpacks:cafa1a0563ae97961d5082d1c9db9a7b33b24f45a040b9124589ab7dce721c46
Starting deploy...
 - service/web configured
 - deployment.apps/web configured
Waiting for deployments to stabilize...
 - deployment/web is ready.
Deployments stabilized in 2.161 seconds
Press Ctrl+C to exit
Watching for changes...
[web] 
[web]   .   ____          _            __ _ _
[web]  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
[web] ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
[web]  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
[web]   '  |____| .__|_| |_|_| |_\__, | / / / /
[web]  =========|_|==============|___/=/_/_/_/
[web]  :: Spring Boot ::        (v2.0.5.RELEASE)
[web] 
[web] 2022-05-23 23:07:01.065  INFO 19 --- [           main] hello.Application                        : Starting Application v0.1.0 on web-5557db7bf7-8qgvc with PID 19 (/workspace/target/hello.jar started by cnb in /workspace)
[web] 2022-05-23 23:07:01.068  INFO 19 --- [           main] hello.Application                        : No active profile set, falling back to default profiles: default
[web] 2022-05-23 23:07:01.110  INFO 19 --- [           main] ConfigServletWebServerApplicationContext : Refreshing org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext@4141d797: startup date [Mon May 23 23:07:01 UTC 2022]; root of context hierarchy
[web] 2022-05-23 23:07:02.227  INFO 19 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
[web] 2022-05-23 23:07:02.258  INFO 19 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
[web] 2022-05-23 23:07:02.259  INFO 19 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet Engine: Apache Tomcat/8.5.34
[web] 2022-05-23 23:07:02.270  INFO 19 --- [ost-startStop-1] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/layers/google.java.maven/maven/lib:/layers/google.java.runtime/java/lib:/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib]
[web] 2022-05-23 23:07:02.348  INFO 19 --- [ost-startStop-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
[web] 2022-05-23 23:07:02.349  INFO 19 --- [ost-startStop-1] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1243 ms
[web] 2022-05-23 23:07:02.410  INFO 19 --- [ost-startStop-1] o.s.b.w.servlet.ServletRegistrationBean  : Servlet dispatcherServlet mapped to [/]
[web] 2022-05-23 23:07:02.414  INFO 19 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'characterEncodingFilter' to: [/*]
[web] 2022-05-23 23:07:02.415  INFO 19 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'hiddenHttpMethodFilter' to: [/*]
[web] 2022-05-23 23:07:02.415  INFO 19 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'httpPutFormContentFilter' to: [/*]
[web] 2022-05-23 23:07:02.415  INFO 19 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'requestContextFilter' to: [/*]
[web] 2022-05-23 23:07:02.545  INFO 19 --- [           main] o.s.w.s.handler.SimpleUrlHandlerMapping  : Mapped URL path [/**/favicon.ico] onto handler of type [class org.springframework.web.servlet.resource.ResourceHttpRequestHandler]
[web] 2022-05-23 23:07:02.733  INFO 19 --- [           main] s.w.s.m.m.a.RequestMappingHandlerAdapter : Looking for @ControllerAdvice: org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext@4141d797: startup date [Mon May 23 23:07:01 UTC 2022]; root of context hierarchy
^CCleaning up...
 - No resources found


skaffold/examples/buildpacks-node$ ../../out/skaffold build
Generating tags...
 - skaffold-buildpacks-node -> skaffold-buildpacks-node:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks-node: Found. Tagging

skaffold/examples/buildpacks-node$ ../../out/skaffold dev
Listing files to watch...
 - skaffold-buildpacks-node
Generating tags...
 - skaffold-buildpacks-node -> skaffold-buildpacks-node:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks-node: Found. Tagging
Tags used in deployment:
 - skaffold-buildpacks-node -> skaffold-buildpacks-node:68df18ae0d599576da595d0de7ab2534e0932fea9698c5dcc0f3aad7a2dae02e
Starting deploy...
 - service/web configured
 - deployment.apps/web configured
Waiting for deployments to stabilize...
 - deployment/web is ready.
Deployments stabilized in 1.113 second
Press Ctrl+C to exit
Watching for changes...
[web] 
[web] > backend@1.0.0 start /workspace
[web] > node src/index.js
[web] 
[web] Example app listening on port 3000!
^CCleaning up...
 - No resources found


skaffold/examples/buildpacks-python$ ../../out/skaffold build
Generating tags...
 - skaffold-buildpacks -> skaffold-buildpacks:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks: Found. Tagging

skaffold/examples/buildpacks-python$ ../../out/skaffold dev
Listing files to watch...
 - skaffold-buildpacks
Generating tags...
 - skaffold-buildpacks -> skaffold-buildpacks:v1.38.0-64-g162022bac
Checking cache...
 - skaffold-buildpacks: Found. Tagging
Tags used in deployment:
 - skaffold-buildpacks -> skaffold-buildpacks:2dd5f27825683ea0761c4a0552959e8dec9eea33c184a3d1e56e6769e7d9e9eb
Starting deploy...
 - service/web configured
 - deployment.apps/web configured
Waiting for deployments to stabilize...
 - deployment/web is ready.
Deployments stabilized in 2.147 seconds
Press Ctrl+C to exit
Watching for changes...
[web]  * Serving Flask app 'web.py' (lazy loading)
[web]  * Environment: production
[web]    WARNING: This is a development server. Do not use it in a production deployment.
[web]    Use a production WSGI server instead.
[web]  * Debug mode: off
[web]  * Running on all addresses (0.0.0.0)
[web]    WARNING: This is a development server. Do not use it in a production deployment.
[web]  * Running on http://127.0.0.1:8080
[web]  * Running on http://172.17.0.4:8080 (Press CTRL+C to quit)
^CCleaning up...
 - No resources found
 ```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -226,6 +227,7 @@ require (
 	github.com/prometheus/client_model v0.2.0 // indirect
 	github.com/prometheus/common v0.31.1 // indirect
 	github.com/prometheus/procfs v0.7.3 // indirect
+	github.com/rakyll/statik v0.1.7 // indirect
```
Comment (aaron-prindle) [inline go.mod:230]: we should no longer need `statik`

Comment (kadern0) [reply go.mod:230]: Hi @aaron-prindle , this is what happened when I tried to run the tests:
```
skaffold$ make test
go test -count=1 -race -short -timeout=90s ./pkg/skaffold/... ./cmd/... ./hack/... ./pkg/webhook/...
cmd/skaffold/app/cmd/statik/statik.go:7:2: no required module provides package github.com/rakyll/statik/fs; to add it:
	go get github.com/rakyll/statik/fs

=== Slow Tests ===
exit status 1
make: *** [Makefile:112: test] Error 1

skaffold$ go get github.com/rakyll/statik/fs
go: added github.com/rakyll/statik v0.1.7

skaffold$ make test
go test -count=1 -race -short -timeout=90s ./pkg/skaffold/... ./cmd/... ./hack/... ./pkg/webhook/...
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/apiversion	0.184s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/bazel	0.399s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/buildpacks	0.573s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/custom	0.316s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/docker	0.764s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache	0.503s
ok  	github.com/GoogleContainerTools/skaffold/pkg/skaffold/build	1.052s
           ...
```

Am I missing something?

**Comment by aaron-prindle** [comment]
@kadern0 I think the issue might be that you are not fully up-to-date/rebased onto master (this was a recent change - https://github.com/GoogleContainerTools/skaffold/pull/7339

Can you try rebasing and see if it is still necessary?

**Comment by kadern0** [comment]
I did rebase before making this change but it seems something didn't go as expected. I'll give it another go (no pun intended) and I'll look specifically for the "statick" module.

File: go.mod (hunk 1 above)
Comment (aaron-prindle) [reply go.mod:230]: Im a bit confused why that error occurred as statik.go was removed
https://github.com/GoogleContainerTools/skaffold/pull/7339/files#diff-1b85c990747d535d0dc3f220994e9d1eb32662ebe38aa052e60bc15386763d71

```
aprindle@aprindle ~/skaffold  [main]$ find . -name statik.go # no output
```

**Comment by kadern0** [comment]
@aaron-prindle, I did a new `fetch upstream` + `rebase upstream/main` and I'm getting the same error. Any idea what could be going on?

This is the last commit on the commit log:

```
commit 162022bac7a77cde755de53d7667d8ec26558f36 (HEAD -> main, upstream/main, origin/main)
Author: ericzzzzzzz <102683393+ericzzzzzzz@users.noreply.github.com>
Date:   Mon May 23 13:11:22 2022 -0700

    chore: revert test runMode, uncomment debug_test/helm (#7432)
```

**Comment by aaron-prindle** [comment]
Not quite sure, that is the correct last commit for `main`.  If you can just remove that line and submit again, the CI will run `make test`.  I'm not sure why your local copy seems to have this file (it's not in the commits or in your PR here)

**Review by aaron-prindle — APPROVED** [review]
LGTM! Thanks @kadern0!

**Pull request merged**